### ISSUE TITLE
Cannot find module 'parse-server-simple-mailgun-adaptor'

### ISSUE THREAD

**Issue by jferrer21** [parse-community/parse-server#4535]
Parse Server 2.7.2

Error when running the index.js file.

Error: Cannot find module 'parse-server-simple-mailgun-adaptor'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at loadAdapter (/home/ubuntu/parse-server-example/node_modules/parse-server/lib/Adapters/AdapterLoader.js:27:15)
    at loadAdapter (/home/ubuntu/parse-server-example/node_modules/parse-server/lib/Adapters/AdapterLoader.js:34:12)
    at getUserController (/home/ubuntu/parse-server-example/node_modules/parse-server/lib/Controllers/index.js:160:65)
    at Object.getControllers (/home/ubuntu/parse-server-example/node_modules/parse-server/lib/Controllers/index.js:88:26)
    at new ParseServer (/home/ubuntu/parse-server-example/node_modules/parse-server/lib/ParseServer.js:136:40)
    at new _ParseServer (/home/ubuntu/parse-server-example/node_modules/parse-server/lib/index.js:54:18)

**Comment by dplewis** [comment]
Looks like its spelt wrong *adapter*

**Comment by jferrer21** [comment]
Thank you for the hint.  I looked to an old installation of Parse_Server and I noticed that we have a module named parse-server-simple-mailgun-adapter under node_modules, however, in version 2.7.2 that module doesn't exist and now what we have is under node_modules -> @ parse -> simple-mailgun-adaptor.  I changed my index file to:

`emailAdapter: {
    module:'@parse/simple-mailgun-adapter',`  

And, it compiled ok.  I haven't done any tests yet.
